_pages/code-of-conduct.md

```diff
@@ -96,27 +96,27 @@ Admins and other leaders who do not follow or enforce the Code of Conduct in goo
 
 The vast majority of situations work out like this, in our experience. This interaction is common, and generally positive.
 
-> Alex: "Yeah I used X and it was really crazy!"
+> Alex: "Yeah I used X and it was [negative association with mental health]"
```

Comment:
Many many people wouldn't know that kind of words that would involve, so could we link to this document: https://wealljs.org/rfc-slackbot-language-shorthands?

I know that you don't want us to use these words in our documentation (i agree), but people need to know what we dont tolerate and mental health slurs are so ingrained in our culture.